lib/chef/knife/bootstrap/bootstrapper.rb

```diff
@@ -348,8 +350,6 @@ def get_chef_extension_private_params
               key_path = create_node_and_client_pem
               pri_config[:client_pem] = File.read(key_path)
```

Comment:
As we now fall through to using the Client pem I think we should also check the client_pem exists and put out a message if we can't find either key.  This would help debugging.

Reply:
@stuartpreston  this line of code itself throws and error before we assign that 
```
ERROR: I cannot read /home/msys/workspace/chef-repo/.chef/client.pem, which you told me to use to sign requests!
````
https://github.com/chef/knife-azure/blob/7b7029b0dcf07530e85c626465f297ef6176163e/lib/chef/knife/bootstrap/bootstrapper.rb#L326

so we don't require to validate again. Let me know your thoughts.

Reply:
@stuartpreston yes but we can have additional check for if the client.pen generated by client_builder exist on the given path (which is a key_path here). Updated the same. Thank you.